README.md.mustache

```diff
@@ -84,7 +84,7 @@ is via [OPAM](https://opam.ocaml.org/doc/Install.html):
 
 ```shell
 opam repo add coq-released https://coq.inria.fr/opam/released
-opam install coq-{{ shortname }}
+opam install {{# opam-name}}{{ opam-name }}{{/ opam-name}}{{^ opam-name}}coq-{{ shortname }}{{/ opam-name}}
```

Comment:
I guess this is fine, but isn't it a better idea to simply change the name of the repo to something that is _not_ "coq-<name>"? Recall that there is redirection in place when changing name.

Reply:
Yes, I've considered this. Here is what made me introduce this change: the "From Sets to Bits in Coq" paper by A. Blot, P.-E. Dagand, J. Lawall(2016) mentions `coq-bits` library, but does not provide a direct link. So I'm sure if a maintainer should change a library's name. On the other hand I don't have a strong opinion on this.

Reply:
Ok, I'm having second thoughts about this: `coq-bits` is called `Bits` when importing it in Coq, so I agree with your proposal to rename the project repo. Since that was the primary motivation behind this PR, I think it could be closed then.